server/pbench/bin/pbench-move-unpacked

```diff
@@ -44,15 +44,21 @@ linkdest=MOVED-UNPACKED
 echo $TS
 
 # get the list of files we'll be operating on - sort them by size
-list=$UNPACK_PATH/pbench-unpack-tarballs.$$
 tmp=$TMP/$PROG.$$
+sort_list=$tmp/sorted.list
 mail_content=$tmp/mail_content.log
-find $ARCHIVE/*/$linksrc -name '*.tar.xz' -printf "%s\t%p\n" 2>/dev/null | grep -v DUPLICATE | sed 's/\/'$linksrc'//' | sort -n > $list
 
-trap 'rm -rf $list $tmp' EXIT INT QUIT
+trap 'rm -rf $tmp' EXIT INT QUIT
 
 mkdir -p $tmp
 
+tar_list=$(find $ARCHIVE/*/$linksrc -name '*.tar.xz' 2>/dev/null | grep -v DUPLICATE | sed 's/\/'$linksrc'//')
+for list in $tar_list; do
+    ref_tar=`readlink -f $list`
+    tar_size=$(stat -c%s "$ref_tar")
+    echo $tar_size $ref_tar
+done | sort -n > $sort_list
```

Comment:
The variable "list" is misnamed: it's not a list, it's just a single link. But more importantly, I think this can be done in aoo pipeline which will be much faster than a for-loop in bash (and will save you from creating extra variables).